src/normal.c

```diff
@@ -3323,6 +3323,12 @@ nv_ctrlo(cmdarg_T *cap)
     static void
 nv_hat(cmdarg_T *cap)
 {
+    if (curwin->w_p_wfb)
+    {
+        semsg("%s", e_winfixbuf_cannot_go_to_buffer);
+        return;
```

Comment:
and here please

Reply:
In retrospect, we can probably remove this because of the check in `getfile` and consolidate

Reply:
yes, please consolidate